mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/intercepting/LoadingProcessActivePredicate.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.intercepting;
+
+import java.util.Objects;
+
+/**
+ * Represents a predicate (boolean-valued function) of active predicate.
+ *
+ * <p>
+ * This is a <a href="package-summary.html">functional interface</a>
+ * whose functional method is {@link #test()}.
+ *
+ */
+public interface LoadingProcessActivePredicate {
```

Comment:
Once the `active()` methods are removed in the various contexts, this interface can be removed.